File: src/main/java/com/my/admin/app/commerce/model/api/response/UserResponse.java
package com.my.admin.app.commerce.model.api.response;

import com.fasterxml.jackson.annotation.JsonInclude;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

@Data
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class UserResponse {

    @ApiModelProperty(value = "User Id", example = "1")
    private Long id;

    @ApiModelProperty(value = "User full name", example = "Full Name")
    private String fullName;

    @ApiModelProperty(value = "User name", example = "username")
    private String username;
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/model/api/response/UserResponse.kt
package com.bk.reactive.app.commerce.admin.my.model.api.response

import com.fasterxml.jackson.annotation.JsonInclude
import io.swagger.annotations.ApiModelProperty

@JsonInclude(JsonInclude.Include.NON_EMPTY)
class UserResponse(
    @ApiModelProperty(value = "User Id", example = "1")
    val id: String,

    @ApiModelProperty(value = "User full name", example = "Full Name")
    val fullName: String,

    @ApiModelProperty(value = "User name", example = "username")
    val username: String
)